### PR TITLE
Docs: add new example to the README

### PR DESCRIPTION
**Pull request by josepot**
I'm not sure about this, but I'm thinking about adding a kinda complete example on the README until the first version of the docs is ready... What do you think @voliva ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +50,113 @@ function App() {
...
+    </div>
+  )
+}
+
+export default function App() {
+  return (
+    <>
+      <Header>Search Github Repos</Header>
+      <Search onSubmit={onSubmit} />
+      <Suspense fallback={() => <LoadingResults />}>
```
Comment (voliva) [inline README.md:152]: The fallback should be an element. I also see there's the same mistake in the temporary Docs section.

File: README.md [hunk 2]
```diff
@@ -29,10 +29,6 @@ Main features
 - :microscope: Tiny and tree-shakeable
 - :muscle: Supports TypeScript
 
-## Examples
-
-- [This is a contrived example](https://codesandbox.io/s/crazy-wood-vn7gg?file=/src/fakeApi.js) based on [this example](https://reactjs.org/docs/concurrent-mode-patterns.html#reviewing-the-changes) from the React docs.
-
 ## Docs
 ```tsx
 const [useDocs] = connectObservable(NEVER)
```
Comment (voliva) [inline README.md:34]: Now that I think of, ☝️ this kinda reads as docs will NEVER come :P

File: README.md (hunk 1 above)
Comment (josepot) [reply README.md:152]: :scream:

File: README.md (hunk 2 above)
Comment (josepot) [reply README.md:34]: Yep... I'm not sure if ppl will find that funny :grimacing: So, we better hurry up and finish the first version of the docs.

**Pull request merged**